MAINTAINERS.md

```diff
@@ -0,0 +1,144 @@
+This section of the document can and should be updated as the above considerations fluctuate. Changes to this section of the document fall under the same requirements as other sections. When changing this section, maintainers must re-review and agree with the document in its entirety, as other guidelines (e.g. voting requirements) will likely change as a result.
+
+### Changes in Maintainership
+SPIRE maintainers are appointed according to the [process described in the governance document][2]. Maintainers may voluntarily step down at any time. Unseating a maintainer against their will requires a unanimous vote with the exception of the unseated.
+
+Unseating a maintainer is an extraordinary circumstance. A process to do so is necessary, but its use is not intended. Careful consideration should be made when voting in a new maintainer, particularly in validating that they pledge to uphold the terms of this document. To ensure that these decisions are not taken lightly, and to maintain long term project stability and foresight, no more than one maintainer can be involuntarily unseated in any given nine month period.
+
+The CNCF MUST be notified of any changes in maintainership via the CNCF Service Desk.
+
+#### Onboarding a New Maintainer
+New SPIRE maintainers participate in an onboarding period during which they fulfill all code review and issue management responsibilities that are required for their role. The length of this onboarding period is variable, and is considered complete once both the existing maintainers and the to-be-appointed maintainer are comfortable. This process MUST be completed prior to the individual in question being named an official SPIRE maintainer.
+
+The onboarding period is intended to ensure that the to-be-appointed maintainer is able/willing to take on the time requirements, familiar with SPIRE core logic and concepts, understands the overall system architecture and interactions that comprise it, and is able to work well with both the existing maintainers and the community.
+
+## Change Review and Disagreements
+The SPIRE project abides by the same [change review process][3] as the SPIFFE project, unless otherwise specified.
+
+The exact definition/difference between "major" and "minor" changes is left to maintainer's discretion. Changes to particularly sensitive areas like the agent's cache manager, or the server's CA, are always good candidates for additional review. If in doubt, always ask for another review.
+
+If there is a disagreement amongst maintainers over a contribution or proposal, a vote may be called in which a simple majority wins. If any maintainer feels that the result of this vote critically endangers the project or its users, they have the right to raise the matter to the SPIFFE TSC. If this occurs, the contribution or proposal in question MUST be frozen until the SPIFFE TSC has made a decision. Do not take this route lightly (see [General Governance](general-governance)).
+
+### Experience Matters
```

Comment:
maybe `User Experience Matters` ?

Reply:
I see multiple points in this section. Perhaps something along the lines of "Preserve accessibility while maintaining security" or "Maintain security, preserve accessibility"?

Reply:
Maybe just "User Experience" or "Security and Usability"?